fast64_internal/sm64/sm64_level_writer.py

```diff
@@ -448,38 +453,44 @@ def removeActSelectorIgnore(basePath, levelEnum):
 	if data != newData:
 		saveDataToFile(filepath, newData)
 
-def parseLevelFast64Ignores(scriptData: str, levelScript: LevelScript):
-	keep: str = None
+areaNumReg = re.compile(r'.*AREA\(([0-9]+),.+\),')
+persistentBlockReg = re.compile(r'.*Fast64 begin persistent block (\[[\w ]+\]).*')
+persistentBlockEndReg = re.compile(r'.*Fast64 end persistent block (\[[\w ]+\]).*')
```

Comment:
```suggestion
persistentBlockBeginReg = re.compile(r'.*' + beginPersistentBlockMagic + ' (\[[\w ]+\]).*')
persistentBlockEndReg = re.compile(r'.*' + endPersistentBlockMagic + ' (\[[\w ]+\]).*')
```
the point of defining a magic variable for a value is to not have it hardcoded in several places 😅

Reply:
😔 

yeahhhhhhh makes sense lol changing it